bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/TTZJets/TTZJetsToLLNuNu_5f_NLO/TTZJetsToLLNuNu_5f_NLO_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model loop_sm-no_b_mass
+
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+
+generate p p > t t~ l+ l- / h [QCD] @0
+add process p p > t t~ vl vl~ / h [QCD] @1
```

Comment:
why not separate it to two processes, this mix will cause difficulty in naming of request, and it's more flexible to set the event number with two processes

Reply:
I don't think it will cause any issue when you have two separate samples instead of one. The advantage of using two processes is that it may also useful for others who only interested in one of the two processes

Reply:
I don't think so that I need to follow about these change because in the last comment according to @sihyunjeon these process are being already initiated to produce. So, I removed to avoid  double card being produced.